### DISCUSSION TITLE
zh/docs/radius/overview

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#3101] [Docs comments]
# zh/docs/radius/overview

如何在Casdoor中连接LDAP客户端

https://casdoor.org/zh/docs/radius/overview

**Comment by yhgroup** [comment]
腾讯radius双因子认证的产品停服想用casdoor替代。防火墙带的VPN没找到配置扩展属性的地方，没办法修改radius下的server.go和util.go，organization为空时默认一个组织机构，同时防火墙不支持OTP方式改成不校验用户密码只校验MFA动态口令，这样防火墙就能用这个radius服务代替腾讯的服务了。
